packages/solo/public/wallet-bridge.html

```diff
@@ -154,11 +154,13 @@
         });
       };
 
-      // Start the flow of messages.
-      if (window.parent !== window) {
-        window.parent.postMessage({ type: 'walletBridgeLoaded' }, '*');
-      }
-      retryWebSocket();
+      window.addEventListener('load', () => {
```

Comment:
your PR comment would make a good code comment.

```suggestion
      // This ensures the message wont be posted until after the iframe's "onLoad" event fires
      // so we can rely on the consistent ordering of events in the WalletConnection component.
      window.addEventListener('load', () => {
```

Reply:
yes, please!

Reply:
Done